src/frontends/tensorflow/src/op/strided_slice.cpp

```diff
@@ -28,16 +28,19 @@ OutputVector translate_strided_slice_op(const NodeContext& node) {
     auto shrink_axis_mask = node.get_attribute<int64_t>("shrink_axis_mask", 0);
 
     auto mask_to_vector = [](int64_t mask) {
-        auto length = sizeof(mask) * CHAR_BIT;
-        vector<int64_t> vec(length, 0);
+        auto max_length = sizeof(mask) * CHAR_BIT;
+        vector<int64_t> vec(max_length, 0);
         if (mask == 0) {
-            return vec;
+            return vector<int64_t>({0});
         }
-        for (auto i = 0; i < length; ++i) {
+        size_t length = 0;
+        for (auto i = 0; i < max_length; ++i) {
             if (static_cast<unsigned char>(mask >> i & 0x1) == 1) {
+                length = i + 1;
                 vec[i] = 1;
             }
         }
+        vec.resize(length);
```

Comment:
What will happen if resulting vector will be shorter then needed? E.g for begin [1,2,3] mask will be [1,1] instead of [1,1,0]?

Reply:
Reverted. Not it is risky to change because different plugins may have different behavior and the specification is not clear